weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -328,6 +338,14 @@ MyApplet.prototype = {
    }
 
   // Override Methods: Applet
+, _onKeySettingsUpdated: function _onKeySettingsUpdated() {
+    if (this.keybinding != null)
+      Main.keybindingManager.addHotKey(UUID,
```

Comment:
As a matter of style, I would prefer braces around this statement since it spans multiple lines.